src/frontend/src/utils/condition.rs

```diff
@@ -73,50 +72,30 @@ impl Condition {
         self.conjunctions.is_empty()
     }
 
-    pub fn to_expr(self) -> ExprImpl {
-        let mut iter = self.conjunctions.into_iter();
-        if let Some(mut ret) = iter.next() {
-            for expr in iter {
-                ret = FunctionCall::new(ExprType::And, vec![ret, expr])
-                    .unwrap()
-                    .into();
-            }
-            ret
-        } else {
-            Literal::new(Some(ScalarImpl::Bool(true)), DataType::Boolean).into()
-        }
+    pub fn into_expr(self) -> ExprImpl {
```

Comment:
What about `impl Into<ExprImpl> for Condition`?

Reply:
How about `impl From<Condition> for ExprImpl` and `impl From<'a &Condition> for ExprImpl` ?